mina.netty/src/test/java/org/kaazing/mina/netty/NioDatagramChannelIoAcceptorIT.java

```diff
@@ -56,16 +55,20 @@
     @Rule
     public TestRule timeout = new DisableOnDebug(new Timeout(10, TimeUnit.SECONDS));
 
-    private IoAcceptor acceptor;
+    private NioDatagramChannelIoAcceptor acceptor;
     private DatagramSocket socket;
 
     @Before
     public void initResources() throws Exception {
-        DatagramChannelIoSessionConfig sessionConfig = new DefaultDatagramChannelIoSessionConfig();
+        DatagramChannelIoSessionConfig sessionConfig = new DefaultNioDatagramChannelIoSessionConfig();
         sessionConfig.setReuseAddress(true);
+        sessionConfig.setReceiveBufferSize(65536);
+        sessionConfig.setSendBufferSize(65536);
         NioWorkerPool workerPool = new NioWorkerPool(newCachedThreadPool(), 4);
         NioServerDatagramChannelFactory channelFactory = new NioServerDatagramChannelFactory(newCachedThreadPool(), 1, workerPool);
         acceptor = new NioDatagramChannelIoAcceptor(sessionConfig, channelFactory);
+        acceptor.setReceiveBufferSize("65536");
+        acceptor.setSendBufferSize("65536");
         acceptor.getFilterChain().addLast("logger", new LoggingFilter());
```

Comment:
Is there a way we can check these buffer sizes are actually used? Generally we need a way to verify the code changes in this PR.